config-templates/redhat/odf-local/changelog.json

```diff
@@ -1,4 +1,21 @@
 [
+    {
+        "4.16": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "12 October 2024",
+                "cves": "",
+                "golang": "1.22.1",
```

Comment:
are you using common golang base image to build ODF images? If yes, please confirm if golang is `1.22.1` or `1.22.7`